src/platform/plugins/shared/dashboard/server/dashboard_saved_object/dashboard_saved_object.ts

```diff
@@ -69,8 +71,23 @@ export const createDashboardSavedObjectType = ({
         create: dashboardAttributesSchemaV2,
       },
     },
+    3: {
+      changes: [
+        {
+          type: 'mappings_addition',
+          addedMappings: {
+            sections: { properties: {}, dynamic: false },
+          },
+        },
+      ],
+      schemas: {
+        forwardCompatibility: dashboardAttributesSchemaV3.extends({}, { unknowns: 'ignore' }),
+        create: dashboardAttributesSchemaV3,
+      },
+    },
   },
   mappings: {
+    dynamic: false,
```

Comment:
@elastic/kibana-core Is it safe to add this without some sort of `changes` addition? Basically, we want the dashboard saved object to be **non-dynamic** so that optional attributes no longer require mappings additions - which is what caused this whole mess in the first place.